mkrules.cmake

```diff
@@ -158,16 +163,19 @@ set(LV_INCLUDE
 
 set(LV_SRC
     ${LV_MP}
-
-    ${LV_BINDINGS_DIR}/driver/esp32/espidf.c
-    ${LV_BINDINGS_DIR}/driver/esp32/modrtch.c
-    ${LV_BINDINGS_DIR}/driver/esp32/sh2lib.c
-
     ${LV_PNG}
     ${LV_PNG_C}
     ${LV_BINDINGS_DIR}/driver/png/mp_lodepng.c
-
-    ${LV_ESPIDF}
 )
 
+if(ESP_PLATFORM)
+    set(LV_SRC
```

Comment:
On ESP32 you are setting `LV_SRC` twice, I think you need to append to the list.

Reply:
It is `set(LV_SRC ${LV_SRC} ...)` so it appends, would `list(APPEND LV_SRC ...)` be more appropriate?

Reply:
Both would work. I think `list(APPEND` might be clearer.

Reply:
I changed to `LIST(APPEND ...)`, it is more readable.